README.md

```diff
@@ -2,176 +2,13 @@
 
 ## About
 
-This package is a thin wrapper around [TinyMCE](https://github.com/tinymce/tinymce) to make it easier to use in a React application. If you need detailed documentation on TinyMCE itself head to the [TinyMCE Documentation](https://www.tiny.cloud/docs/).
+This package is a thin wrapper around [TinyMCE](https://github.com/tinymce/tinymce) to make it easier to use in a React application.
 
-For some quick demos, check out the [storybook](https://tinymce.github.io/tinymce-react/).
+* If you need detailed documentation on TinyMCE, see: [TinyMCE Documentation](https://www.tiny.cloud/docs/).
+* For the TinyMCE React component Quick Start, see: [TinyMCE Documentation - React Integration](https://www.tiny.cloud/docs/integrations/react/#tinymcereactintegrationquickstartguide).
+* For the TinyMCE React component Technical Reference, see: [TinyMCE Documentation - TinyMCE React technical reference](https://www.tiny.cloud/docs/integrations/react/#tinymcereacttechnicalreference).
```

Comment:
"TinyMCE React component Technical Reference" feels a bit awkward? Though mostly just because of the non-capitalised word amongst a bunch of capitalised words I think, so not sure it's worth doing anything about that.